queue_services/entity-filer/src/entity_filer/filing_processors/filing_components/rules_and_memorandum.py

```diff
@@ -0,0 +1,72 @@
+# Copyright © 2022 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Manages the share structure for a business."""
+from __future__ import annotations
+from tokenize import String
+
+from typing import Dict, List, Optional
+
+from dateutil.parser import parse
+from legal_api.models import Business
+from legal_api.models import Business, Document, Filing
+from legal_api.models.document import DocumentType
+from legal_api.services.minio import MinioService
+from entity_filer.utils import replace_file_with_certified_copy
+
+
+def update_rules(business: Business, filing: Filing, rules_file_key: String, rules_file_name: String) -> Optional[List]:
+    """Updtes rules if any
+    Assumption: rules file key and name have already been validated
+    """
+    if not business or not rules_file_key or not rules_file_name:
+        # if nothing is passed in, we don't care and it's not an error
+        return None
+
+    rules_file = MinioService.get_file(rules_file_key)
+    replace_file_with_certified_copy(rules_file.data, business, rules_file_key, business.founding_date)
```

Comment:
so you've confirmed that the rules and memorandum docs need to be certified?

Reply:
This still neeeds to be re-confimed. Added tag to not merge this PR for now.

Reply:
Yeah certification is required. Checked with Zac

Reply:
ok, thanks for checking